public/js/components/Editor.js

```diff
@@ -363,7 +363,7 @@ const Editor = React.createClass({
 });
 
 module.exports = connect(
-  (state, props) => {
+  state => {
```

Comment:
It comes down to a bug in React 0.14.7, I believe. At some point I'd like to figure out exactly why it's not calling lifecycle methods the same, but at least it's not intentional behavior.